boilerplate/app/i18n/hi.ts

```diff
@@ -0,0 +1,131 @@
+import demoHi from "./demo-hi"
+import { Translations } from "./en"
+
+const hi: Translations = {
+  common: {
+    ok: "ठीक है!",
+    cancel: "रद्द करें",
+    back: "वापस",
+    logOut: "लॉग आउट", // @demo remove-current-line
+  },
+  welcomeScreen: {
+    postscript:
+      "पीएसटी - शायद आपका ऐप ऐसा नहीं दिखता है। (जब तक कि आपके डिजाइनर ने आपको ये स्क्रीन नहीं दी हों, और उस स्थिति में, इसे लॉन्च करें!)",
```

Comment:
Psst is transliterated as P S T. Doesn't seem appropriate.

Reply:
You're right. I've thought of removing "पीएसटी - ", but then it does not retain the casual tone.